src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  width: $iui-baseline * 27;
```

Comment:
baseline should only be used for height/vertical spacing.

And because of how random this is, you have some extra space on the right side.
![image](https://user-images.githubusercontent.com/9084735/130491280-0fc466fb-acd6-4ee7-b448-a3e878ddb78b.png)

For calculating, I would suggest doing this:
1. Decide how many columns you want to show (lets say 8).
2. Add the widths and spaces around those columns (24\*8 + 12\*7 + 16\*2 = 308)
3. Find a large spacing variable and multiply it to get close to the number above (e.g. 64 * 5 = 320)

Reply:
We discussed in a call today, we're going to set a max-width of 10 swatches in the basic color picker.